docs/UserGuide.md

```diff
@@ -23,43 +23,82 @@ Reache is a desktop app that helps busy working professionals manage their large
 
    ![Ui](images/Ui.png)
 
-5. Refer to the [Features](#2-features) below for details of each command.
+5. Refer to the [Features](#3-features) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
 
+## 2. Introduction
+### 2.1 Overview
+Reache is a contact manager application that helps you:
+- Store and organize contacts
+- Easily sort and search through your contact list
+- Plan and keep track of upcoming meetings
+
+<br>
+
+### 2.2 Interface Guide
+#### 2.2.1 Main Window
+Upon launching Reache, you will see the Main Window, which is depicted below. The Main Window displays your entire contact list as well as all upcoming meetings you have planned.
+
+![Main Window](images/MainWindow.png)
```

Comment:
I can't see the images here, but I think that's just how PRs work. Hopefully, it works fine when deployed.